ndcube/global_coords/global_coords.py

```diff
@@ -28,6 +30,21 @@ def _all_coords(self):
         A dynamic dictionary of all global coordinates, stored here or derived
         from the ndcube object.
         """
+        if self._ndcube is None:
+            return self._internal_coords
+
+        if hasattr(self._ndcube.wcs.low_level_wcs, "dropped_world_dimensions"):
+            dropped_world = self._ndcube.wcs.low_level_wcs.dropped_world_dimensions
+            wcs_dropped = {}
+            if dropped_world != {}:
+                for i in range(len(dropped_world["value"])):
+                    name = dropped_world["world_axis_names"][i] or\
+                        dropped_world["world_axis_physical_types"][i]
```

Comment:
I have a personal hatred of line continuation characters lol
```suggestion
                    name = (dropped_world["world_axis_names"][i] or
                            dropped_world["world_axis_physical_types"][i])
```

Reply:
A valid stance!